<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultimeterWidget</class>
 <widget class="QWidget" name="MultimeterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{background-image: url(:/new/prefix1/res/multimeter-sensor/multimeter-background.png);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_T">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>310</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/res/main/button1.png);</string>
    </property>
    <property name="text">
     <string>温度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>110</y>
      <width>30</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>电流</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_P">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/res/main/button1.png);</string>
    </property>
    <property name="text">
     <string>压力</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>95</y>
      <width>500</width>
      <height>170</height>
     </rect>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Close">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>440</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/res/main/button1.png);</string>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_A">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/res/main/button1.png);</string>
    </property>
    <property name="text">
     <string>电流</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
